src/Components/Patient/DailyRounds.tsx

```diff
@@ -788,6 +788,30 @@ export const DailyRounds = (props: any) => {
                   },
                 ]}
               />
+              <RangeAutocompleteFormField
+                {...field("ventilator_spo2")}
+                unit="%"
+                start={0}
+                end={100}
+                step={1}
+                thresholds={[
+                  {
+                    value: 0,
+                    className: "text-danger-500",
+                    label: "Low",
+                  },
+                  {
+                    value: 90,
+                    className: "text-primary-500",
+                    label: "Normal",
```

Comment:
i18n

Reply:
`    thresholds={[
                  {
                    value: 0,
                    className: "text-danger-500",
                    label: t("low"),
                  },
                  {
                    value: 90,
                    className: "text-primary-500",
                    label: t("normal"),
                  },
                  {
                    value: 100,
                    className: "text-danger-500",
                    label: t("high"),
                  },
                ]}
              />`
              
              Is this fine?

Reply:
![Uploading Screenshot from 2024-10-23 10-32-41.png…]()

Reply:
yup

Reply:
![Screenshot from 2024-10-23 10-32-41](https://github.com/user-attachments/assets/af66ffb0-8522-4fe6-b238-c798e3be24fb)